site/guide/get-started-developer-framework.qmd

```diff
@@ -1,174 +1,90 @@
 
-The developer framework is designed to be model agnostic. If your model is built in Python, ValidMind’s Python client library already provides all the standard functionality you might need without requiring your developers to rewrite any functions. 
+The Developer Framework is designed to be model agnostic. If your model is built in Python, ValidMind’s Python client library already provides all the standard functionality you might need without requiring your developers to rewrite any functions.
 
 ::: {.callout-important collapse="true" appearance="minimal"}
-## {{< fa building-columns >}} Key ValidMind concepts
+##  {{< fa angle-right >}} Key ValidMind concepts
 
 <!--- These concepts were already introduced in the "Get started" section. They are repeated here in a collapsed format in case a reader missed them earlier. --->
 
 {{< include key-validmind-concepts.qmd >}}
 
 :::
 
-## How do I use the framework?
-
-A typical high-level workflow for model developers consists of four major steps:
-
-```{mermaid}
-graph LR
-    A[Develop<br>model] --> B[Generate model<br>documentation]
-    B --> C[Refine model<br>documentation]
-    C --> D[Submit for review]
-    C --> B
-```
-<br>
-
-**Develop model**
-: In your existing developer environment, build one or more candidate models that need to be validated. This step includes all the usual activities you already follow as a model developer. 
-
-**Generate model documentation**
-: With the ValidMind Developer Framework, generate automated model documentation and run validation tests. This step includes making use of the automation and testing functionality provided by the framework and uploading the output to the Platform UI. You can iteratively regenerate the documentation as you work though the next step of refining your documentation.
-
-**Refine model documentation**
-: In the ValidMind Platform UI, review the generated documentation and test output. Iterate over the documentation and test output to refine your model documentation. Collaborate with other developers and model validators to finalize the model documentation and get it ready for review. 
-
-**Submit for review**
-: In the ValidMind Platform UI, you submit the model documentation for review which moves the documentation workflow moves to the next phase where a model validator will review it.
-
-## How do I generate model documentation?
-
-<!--- NR How do developers verify the current documentation template? --->
-Before you can use the developer framework, you need to verify that the current documentation template contains all the necessary tests for the model you are developing: 
+## Getting started
 
-- The template might already be sufficient and you only need to run the template within the developer framework to populate documentation. 
+After you [**sign up**](https://app.prod.validmind.ai) for ValidMind to get access, start with our QuickStart or watch the video:
```

Comment:
```suggestion
After you [**sign up**](https://app.prod.validmind.ai) for ValidMind to get access, head to the quickstart guide or watch the video demonstration:
```

Reply:
I edited this to change back to "QuickStart" — we should change this globally if that's the new spelling — and committed the rest.